web/src/runtime.ts

```diff
@@ -1477,6 +1489,41 @@ export class Instance implements Disposable {
     this.cacheMetadata = { ...this.cacheMetadata, ...(list["metadata"] as Record<string, any>) };
   }
 
+  /**
+   * Delete NDArray from cache url.
+   *
+   * @param ndarrayCacheUrl
+   * @param device
+   * @param cacheScope
+   */
+  async deleteNDArrayCache(
```

Comment:
I feel like this function should be outside of `Instance`, just like `hasNDArrayInCache()`, since it is more like a helper function. We can then expose this function in https://github.com/apache/tvm/blob/main/web/src/index.ts#L25, so that tvmjs users can call this function.

Reply:
Received, also plan to move the fetchNDArrayInCache to the export